pytransform3d/test/test_rotations.py

```diff
@@ -2232,3 +2232,23 @@ def test_euler_from_quaternion_edge_case():
     matrix = pr.matrix_from_quaternion(quaternion)
     euler_xyz = pr.extrinsic_euler_xyz_from_active_matrix(matrix)
     assert not np.isnan(euler_xyz).all()
+
+
+def test_norm_angle_precision():
+    # test precision of float32
```

Comment:
it's float64, and not float32

Reply:
I'm sorry for causing a misunderstanding. I'd say it only tests the precision of float32 because I only split the interval into 1e7 equal numbers and each number is about 6.28318531e-07 and it can be represented by float32. If one really wants to test the precision of float64 the interval should be split into 1e16 parts. I didn't do so because it will consume too much memory(~71.1 PiB).